### PR TITLE
ON-36596 # Changed `.ob-list__text-secondary`

### PR DESCRIPTION
**Pull request by Zaxist**

### CODE REVIEW

File: src/styles/list.scss [hunk 1]
```diff
@@ -71,8 +71,13 @@
 .ob-list__text-tertiary,
 .ob-list__text-secondary {
   font-size: 0.8rem;
+}
+.ob-list__text-tertiary {
   color: #757575;
 }
+.ob-list__text-secondary {
+  color: grey;
```
Comment (mymattcarroll) [inline src/styles/list.scss:79]: Should this be using one of the bulma colours?

File: src/styles/list.scss [hunk 2]
```diff
@@ -71,8 +71,13 @@
 .ob-list__text-tertiary,
 .ob-list__text-secondary {
   font-size: 0.8rem;
+}
+.ob-list__text-tertiary {
   color: #757575;
```
Comment (divporter) [inline src/styles/list.scss:76]: $grey is #757575 sorry

File: src/styles/list.scss (hunk 1 above)
Comment (Zaxist) [reply src/styles/list.scss:79]: Idk, task said to change back to `grey` which is what it was before

File: src/styles/list.scss (hunk 2 above)
Comment (divporter) [reply src/styles/list.scss:76]: Essentially the task is to put it back how it was

File: src/styles/list.scss (hunk 1 above)
Comment (Zaxist) [reply src/styles/list.scss:79]: Wait, no it wasn't ir was $grey

File: src/styles/list.scss (hunk 2 above)
Comment (Zaxist) [reply src/styles/list.scss:76]: Got it

File: src/styles/list.scss [hunk 3]
```diff
@@ -71,8 +71,13 @@
...
 }
+.ob-list__text-secondary {
+  color: grey;
+}
```
Comment (divporter) [inline src/styles/list.scss]:
```suggestion
.ob-list__text-secondary, 
.ob-list__text-tertiary {
  font-size: 0.8rem;
  color: $grey;
}
```

**Pull request merged**